src/transformers/pipelines.py

```diff
@@ -2752,6 +3012,14 @@ def _concat_inputs_history(self, inputs: List[List[int]], histories: List[Option
             "model": {"pt": "distilbert-base-cased-distilled-squad", "tf": "distilbert-base-cased-distilled-squad"},
         },
     },
+    "table-question-answering": {
+        "impl": TableQuestionAnsweringPipeline,
+        "tf": TFAutoModelForQuestionAnswering if is_tf_available() else None,
+        "pt": AutoModelForQuestionAnswering if is_torch_available() else None,
+        "default": {
+            "model": {"pt": "nielsr/tapas-base-finetuned-wtq", "tf": "nielsr/tapas-base-finetuned-wtq"},
```

Comment:
wait there is a TF implementation of TAPAS? I don't think so no?

Reply:
No there's isn't (the original implementation was in TF1, but there isn't a Transformers-compatible TF implementation)